components/admin_console/feature_flags.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+
+type Props = {
+    config: Record<string, unknown>;
+};
+
+const FeatureFlags: React.FC<Props> = (props: Props) => {
+    const flags = props.config.FeatureFlags as Record<string, string>;
+    let settings = null;
+    if (flags) {
+        settings = Object.keys(flags).map((ffKey) => (
+            <tr key={ffKey}>
+                <td width='20px'/>
+                <td className='whitespace--nowrap'>{ffKey}</td>
+                <td className='whitespace--nowrap'>{flags[ffKey]}</td>
+            </tr>
+        ));
+    }
+
+    return (
+        <div className='wrapper--admin'>
+            <FormattedAdminHeader
+                id='admin.feature_flags.title'
+                defaultMessage='Feature Flags'
+            />
+            <div className='admin-console__wrapper'>
+                <div className='admin-logs-content admin-console__content'>
+                    <div className={'banner info'}>
+                        <div className='banner__content'>
+                            <FormattedMessage
+                                id='admin.feature_flags.introBanner'
+                                defaultMessage={'Feature flag values displayed here show the status of features enabled on this server. The values here are only for use debugging by the Mattermost support team.'}
```

Comment:
```suggestion
                                defaultMessage={'Feature flag values displayed here show the status of features enabled on this server. The values here are used only for troubleshooting purposes by the Mattermost support team.'}
```

Reply:
0/5 on whether it's "debugging" or "troubleshooting", but we use "troubleshooting" more frequently when referring to diagnostic processes. If this is a new/different process then I'd go with "debugging" (but we may need to add a link to a doc explaining the difference?).

Reply:
Makes sense!